src/navigation-tabs.ts

```diff
@@ -17,3 +17,21 @@ export const highlightNavigationTab = ({
             count,
         },
     });
+
+/**
+ * Request the app to refresh the navigation bars
+ */
+export const refreshNavBar = ({
+    module_id,
+    product_id,
+}: {
+    module_id?: string; // module id used in Visual Modules API
+    product_id?: string; // identifier of the product as it is in Subscribed Products API
```

Comment:
I'm not sure if this is the best way to add parameters that are optional. Feedback is welcomed!